src/bundle/msi_bundle.rs

```diff
@@ -158,6 +171,7 @@ fn set_summary_info(package: &mut Package, package_guid: Uuid, settings: &Settin
     }
     let creating_app = format!("cargo-bundle v{}", crate_version!());
     summary_info.set_creating_application(creating_app);
+    summary_info.set_word_count(2);
```

Comment:
This fix requires the `rust-msi` patch. Without this line, the installer will attempt to copy files from outside the package.